frontend/src/components/pages/quiz/QuizPage.tsx

```diff
@@ -1,8 +1,30 @@
-import { Grid, GridItem } from '@chakra-ui/react';
+import { Box, Grid, GridItem } from '@chakra-ui/react';
 import Layout from '../../layout/Layout';
 import quizzes from '../../../data/quizzes';
 import FileSelection from './FileSelection';
 import Editor from './Editor';
+import Countdown from 'react-countdown';
+
+const renderer = ({ hours, minutes, seconds, completed }) => {
```

Comment:
Timer should be moved to separate component so we can easily integrate it with editor